server/config/environment.go

```diff
@@ -62,6 +64,8 @@ func GetEnvironment() envVars {
 	environment.SESSION_COOKIE_MAX_AGE = 0 // 0 to make it a session cookie
 	environment.SAVE_CONTAINER = false
 	environment.START_TIME = time.Now().Format(time.RFC3339)
+	environment.LOG_PATH = "/installerstore/engine.txt"
```

Comment:
With this we now have three configuration elements that have `/installerstore` hard-coded. How about we add one more where we specify the root storage location?

Reply:
sure